test/CreateHandler/CreateHandlerCommandTest.php

```diff
@@ -43,6 +54,32 @@ private function reflectExecuteMethod()
         return $r;
     }
 
+    private function mockApplication()
```

Comment:
As we are using PHP 7.1 I would suggest to add return type (`Application`?)

Reply:
Sure; do we need to add RTH within a test case, though?

Reply:
We don't need to, but it would be nice, especially for IDEs and writing new test.
I would add docblock with `Application|ObjectProphecy` as it is more accurate, and maybe we should go this way, not RTH.

Reply:
This is... confusing. We can't specify a RTH of `Application`, as we actually return a prophecy. But specifying `ObjectProphecy` as the RTH makes it less clear what we're returning. I'll document it via a docblock annotation instead.